linux_os/guide/system/network/network-firewalld/ruleset_modifications/set_firewalld_default_zone/rule.yml

```diff
@@ -50,6 +50,7 @@ references:
     pcidss4: "1.5.1"
     srg: SRG-OS-000480-GPOS-00227
     stigid@rhel7: RHEL-07-040810
+    stigid@rhel8: RHEL-08-040090
```

Comment:
That is the reasoning behind this change?

If you want automation for this rule I think we should add it to `configured_firewalld_default_deny` verses changing the mapping. That can be done in another PR.

Reply:
To be honest, I am unsure about the distinctions between both rules.
If I'm not mistaken, you draw the distinction with:
- `configured_firewalld_default_deny` approach is to update the zone of the existing interfaces
- `set_firewalld_default_zone` approach is to update the firewalld.conf file to contain `DefaultZone=drop`

Is that understanding right?

If so, perhaps [my other PR](https://github.com/ComplianceAsCode/content/pull/11251) should be split between `configured_firewalld_default_deny` and `set_firewalld_default_zone`. Additionally, a check (probably with SCE because there is no firewalld backend in OpenSCAP to query live interfaces, AFAIK) should be written for configured_firewalld_default_deny.
Finally, as you suggested, the STIG identifier should be present on both rules.

Does that course of action appear reasonable?

Reply:
Sounds reasonable to me. 

I will note that the data streams in RHEL and the releases in this project don't ship with SCE content.

Reply:
After thinking about this a bit further, I think adding a remediation "by default" for these rules is quite dangerous, as the warning in https://github.com/ComplianceAsCode/content/blob/8c3e9d6c127522e28d9ed7010476139c01fd8d37/linux_os/guide/system/network/network-firewalld/ruleset_modifications/set_firewalld_default_zone/rule.yml#L62-L67 rightly indicates. Given the risks involved, adding remediations for them seems a bit reckless on my part.

For now, I will simply enable both rules in the stig profile, with `configured_firewalld_default_deny` having no test/remediation, and `set_firewalld_default_zone ` being tested but not remediated.

I will circle back with colleagues next week to determine what could be a proper course of action for this specific pair of rules.

As for the lack of SCE content in official RHEL releases of CAC, this is not a problem for us as we apply multiple configuration on top of the STIG profile and thus were already rebuilding CAC. However, that will indeed be a limitation of said remediation, should we agree on going toward that route.